litellm/utils.py

```diff
@@ -4531,6 +4531,36 @@ def exception_type(
                             llm_provider="palm",
                             response=original_exception.response
                         )
```

Comment:
we also need a block for getting optional params for `gemini`

example here for palm
https://github.com/BerriAI/litellm/blob/707cd0364e10344d38ff621225a264e5ea987ea4/litellm/utils.py#L2463

Reply:
Done added that now.